starcounter-include.html

```diff
@@ -288,6 +289,60 @@
             return range.createContextualFragment(htmlStr);
         }
 
+        /**
+         * Replaces all <slot> tags with <content>,
+         *  preserves all attributes, translates selector to slot name
+         * @param  {String} string stringified HTML
+         * @return {String}        translated HTML
+         */
+        function replaceSlotsWithContentInString(string){
+            return string.replace(/<\/slot(\s*)>/gi,'</content$1>')
```

Comment:
left side white space before tag name in the closing tag is valid HTML, I suggest adding another `\s*` to become `string.replace(/<\/\s*slot(\s*)>/gi,'</content$1>')`.